src/styles/blocks/recommended.scss

```diff
@@ -0,0 +1,70 @@
+.recommended {
+  margin-bottom: 120px;
+
+  @include onTablet {
+    margin-bottom: 128px;
+  }
+
+  &__title {
+    margin-bottom: 48px;
+
+    @include title;
+
+    @include onTablet {
+      margin-bottom: 64px;
+    }
+  }
+
+  &__content {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+
+    @include onDesktop {
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
+      column-gap: 24px;
+    }
+  }
+
+  &__card:not(:last-child) {
+    margin-bottom: 48px;
```

Comment:
Consider adding a comment explaining the purpose of this selector 'not(:last-child)', it will help in understanding the code more easily. 📝